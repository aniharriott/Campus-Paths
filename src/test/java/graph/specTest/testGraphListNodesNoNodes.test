# Test Graph listNodes no nodes file.
# Tests that no nodes are printed
# when there are no nodes in the graph.

# Create graph
CreateGraph graph1

# Print nodes in the graph
ListNodes graph1