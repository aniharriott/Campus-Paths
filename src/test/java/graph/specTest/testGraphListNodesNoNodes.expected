# Test Graph listNodes no nodes file.
# Tests that no nodes are printed
# when there are no nodes in the graph.

# Create graph
created graph graph1

# Print nodes in the graph
graph1 contains: